packages/components/.storybook/config.tsx

```diff
@@ -34,8 +33,10 @@ const GlobalStyle = createGlobalStyle`
   ${global};
   body {
     font-family: 'Inter', sans-serif;
-    background-color: ${theme.colors.sideBar.background};
-    color: ${theme.colors.sideBar.foreground};
+    margin: 0;
+    background-color: ${props =>
+      props.theme.colors.sideBar.background} !important;
+    color: ${props => props.theme.colors.sideBar.foreground} !important;
```

Comment:
its missing the border-box thing

```
const GlobalStyle = createGlobalStyle`
  ${global};
  html body {
    font-family: 'Inter', sans-serif;
    background-color: ${theme.colors.sideBar.background};
    color: ${theme.colors.sideBar.foreground};
    margin: 0;
    padding: 20px;

    * {
      box-sizing: border-box;
    }
  }
`;
```

Reply:
ah yes, will merge with master and bring this along

Reply:
added box-sizing, removed padding though. Do we need it on body?